pool/web/handler/admin_session.ml

```diff
@@ -106,14 +121,19 @@ let new_helper req page =
              flash_fetcher
            |> Lwt_result.ok
          | `New ->
```

Comment:
How about adding the the next match statement directly in the parent one as "when ..."

```suggestion
         | `New when CCOption.is_some experiment.Experiment.online_study ->
           ...
         | `New ->
```

Reply:
It seems that does not work with polymorphic variants as it does with variant types. The compiler wants me to handle the case of `| New -> ` which does not exist.

Reply:
Do we have this separation multiple times? should a ` type handleCreateAs = New | FollowUp` be defined?

Reply:
There are only 2 pattern matches.